### DISCUSSION TITLE
Error in line 231

### DISCUSSION THREAD

**Discussion by loelismurf** [liedekef/events-made-easy#270] [Q&A] [answered]
Hello lovely community,
i got a error Message from wordpress core looks like.
How i can debug, or find the error?
The message Repeat it daily, so i do not find where the probleme are?

Information oft of the email:
Orginal
Wenn Sie Hilfe bei diesem Problem suchen, werden Sie möglicherweise nach einigen der folgenden Informationen gefragt:
WordPress-Version 6.3
Aktives Theme: GeneratePress (Version 3.3.0)
Aktuelles Plugin: Events Made Easy (Version 2.4.5)
PHP-Version 7.4.33

Fehler-Details
==============
Ein Fehler vom Typ E_PARSE wurde in der Zeile 231 der Datei /var/www/vhosts/[Hostername]/httpdocs/wp-content/plugins/events-made-easy/eme-filters.php verursacht. Fehlermeldung: syntax error, unexpected ':', expecting ')'

Translated by deepL:
If you are looking for help with this issue, you may be asked for some of the following information:
WordPress version 6.3
Active theme: GeneratePress (version 3.3.0)
Current plugin: Events Made Easy (version 2.4.5)
PHP version 7.4.33

Error details
==============
An error of type E_PARSE was caused on line 231 of file /var/www/vhosts/[hostname]/httpdocs/wp-content/plugins/events-made-easy/eme-filters.php. Error message: syntax error, unexpected ':', expecting ')'.

**Comment by liedekef** [comment, marked as answer]
Like the plugin readme specifies: PHP 8.0 is required. PHP 7.4 is end of life since beginning of 2023, so I no longer support it.
